packages/app/src/app/components/StripeMessages/SandboxLimitation.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { MessageStripe } from '@codesandbox/components';
+import { useAppState } from 'app/overmind';
+import { useWorkspaceLimits } from 'app/hooks/useWorkspaceLimits';
+import { useScopedPersistedState } from 'app/hooks/usePersistedState';
+
+const SANDBOX_INCREASE_LIMIT_REQUEST_FORM =
+  'https://webforms.pipedrive.com/f/cB3DfL4Vg41gVPKuA6MLRpr8YI6L8KCzxiulfpjgnBdFBxa4xkJpqMSE6g8yHTjjtF';
+const SANDBOX_LIMIT_ANNOUNCEMENT = 'https://codesandbox.io';
```

Comment:
I guess this is a temp link, right? Will it point to a blogpost?

Reply:
yes, waiting for the blogpost :) chicken-egg problem

Reply:
I have it now 😄 
https://www.codesandbox.community/c/api-billing-updates/upcoming-pricing-billing-changes